[Flutter] Doc update on Coinbase Wallet section due to an issue on their side (#1614)
WalletConnect/walletconnect-docs: docs/appkit/flutter/core/installation.mdx

### Enable Coinbase Wallet

:::info VERSION DEPENDENCY

Coinbase Wallet is available from Web3Modal **version 3.1.0** and higher

:::

Since Coinbase Wallet uses it's own SDK there are a few extra but simply steps to do if you are planning to include and support it

:::caution IMPORTANT

There is currently a bug in Coinbase's MobileWalletProtocol which fix is still under review in the following PR https://github.com/MobileWalletProtocol/wallet-mobile-sdk/pull/327 so, until this PR is merged (or the bug is fixed in any way) and a new version of their SDK is published, you will need to override `coinbase_wallet_sdk` in your **pubspec.yaml** as follows if you want to support their wallet:

```dart
dependency_overrides:
  coinbase_wallet_sdk:
    git:
      url: https://github.com/WalletConnect/wallet-mobile-sdk.git
      path: flutter
      ref: 9194f04dbaf41c12b8121f0a5f146cc7faf6b847
```

> If you don't feel to override the dependency you can disable Coinbase Wallet support as explained in the [Disable Coinbase Wallet](#disable-coinbase-wallet) section.

Reviewer: ```suggestion
Alternatively, you can disable Coinbase Wallet support as explained in the [Disable Coinbase Wallet](#disable-coinbase-wallet) section.
```
